### DISCUSSION TITLE
Must-Have Features (working...)

### DISCUSSION THREAD

**Discussion by osman-kocak** [inspireui/support#5484] [Working on Features]
**New design and feature suggestions**

<img width="328" alt="Screen Shot 2022-01-12 at 17 56 44" src="https://user-images.githubusercontent.com/57155882/150650356-e79f7498-5eb3-4d99-882d-4fa5ae9b43d4.png">
<img width="215" alt="Screen Shot 2022-01-12 at 17 57 05" src="https://user-images.githubusercontent.com/57155882/150650358-eee7520a-7971-4198-a13e-d0d2013d2840.png">
<img width="252" alt="Screen Shot 2022-01-12 at 17 58 13" src="https://user-images.githubusercontent.com/57155882/150650360-77e885b4-e0b7-4d02-8c31-720bb3d92832.png">
<img width="208" alt="Screen Shot 2022-01-12 at 17 58 55" src="https://user-images.githubusercontent.com/57155882/150650362-3c17b2bb-e702-4149-a4c1-364b3893ce5f.png">
<img width="203" alt="Screen Shot 2022-01-12 at 17 58 59" src="https://user-images.githubusercontent.com/57155882/150650364-bdb01f32-2c49-4ea9-a1ff-0b2026f4e1b1.png">
<img width="236" alt="Screen Shot 2022-01-12 at 17 59 07" src="https://user-images.githubusercontent.com/57155882/150650365-07bc9455-b3fa-4151-85af-40f1ee8b5948.png">
<img width="228" alt="Screen Shot 2022-01-12 at 17 59 28" src="https://user-images.githubusercontent.com/57155882/150650368-bcdfa3ca-f1bc-4681-b75d-46f2c7b4c4ca.png">
<img width="220" alt="Screen Shot 2022-01-12 at 17 59 43" src="https://user-images.githubusercontent.com/57155882/150650371-c08a231b-f44b-443b-9a30-8be98784ed6a.png">
<img width="208" alt="Screen Shot 2022-01-12 at 18 00 39" src="https://user-images.githubusercontent.com/57155882/150650373-3cf1d402-ddfd-4faf-9938-9aa4aa175ac5.png">
<img width="268" alt="Screen Shot 2022-01-22 at 19 49 57" src="https://user-images.githubusercontent.com/57155882/150650375-51dc9c17-b047-48d5-8ad7-6d7bf1df5f2f.png">
<img width="228" alt="Screen Shot 2022-01-22 at 19 53 24" src="https://user-images.githubusercontent.com/57155882/150650378-ed79e327-529b-4e18-92b1-5ebbc7efbb0d.png">
<img width="252" alt="Screen Shot 2022-01-22 at 19 53 55" src="https://user-images.githubusercontent.com/57155882/150650379-fb6a781c-e145-4e82-b7f6-7394659f786c.png">
<img width="277" alt="Screen Shot 2022-01-22 at 19 56 03" src="https://user-images.githubusercontent.com/57155882/150650381-4e78a2b4-bb52-498a-bb17-59a92c445147.png">
<img width="236" alt="Screen Shot 2022-01-22 at 19 58 18" src="https://user-images.githubusercontent.com/57155882/150650384-7731f8aa-6f4d-4554-8bb0-f7358eb9b0b9.png">
<img width="237" alt="Screen Shot 2022-01-22 at 19 58 37" src="https://user-images.githubusercontent.com/57155882/150650385-5aefb327-4b1b-42b7-a046-5c9bca1d7b1a.png">
<img width="290" alt="Screen Shot 2022-01-22 at 19 58 58" src="https://user-images.githubusercontent.com/57155882/150650387-302b729b-8643-485b-934b-bcb0a4fb19fb.png">
<img width="193" alt="Screen Shot 2022-01-22 at 20 01 03" src="https://user-images.githubusercontent.com/57155882/150650388-7a4e850b-6ad9-41ce-bd02-a078491b6037.png">
<img width="198" alt="Screen Shot 2022-01-22 at 20 01 10" src="https://user-images.githubusercontent.com/57155882/150650389-f6abb89d-ae81-4c1a-add4-e8d2b56788fe.png">
<img width="246" alt="Screen Shot 2022-01-22 at 20 03 39" src="https://user-images.githubusercontent.com/57155882/150650392-f404ffcf-424f-4c65-9810-bef460703367.png">
<img width="186" alt="Screen Shot 2022-01-22 at 20 04 11" src="https://user-images.githubusercontent.com/57155882/150650394-c9c46cee-a9b2-4130-a0d6-de1aa601b6de.png">
<img width="239" alt="Screen Shot 2022-01-22 at 20 04 36" src="https://user-images.githubusercontent.com/57155882/150650395-38aa70e3-6eac-4355-b0f3-530e164895e4.png">
<img width="244" alt="Screen Shot 2022-01-22 at 20 07 29" src="https://user-images.githubusercontent.com/57155882/150650397-a8add2a2-a3bf-4b42-8436-114c35614eb9.png">

**Comment by ecomdeveloperbd** [comment]
https://user-images.githubusercontent.com/57155882/150650375-51dc9c17-b047-48d5-8ad7-6d7bf1df5f2f.png
https://user-images.githubusercontent.com/57155882/150650373-3cf1d402-ddfd-4faf-9938-9aa4aa175ac5.png
https://user-images.githubusercontent.com/57155882/150650381-4e78a2b4-bb52-498a-bb17-59a92c445147.png
https://user-images.githubusercontent.com/57155882/150650388-7a4e850b-6ad9-41ce-bd02-a078491b6037.png

Please add these features ..Recommended

**Reply by osman-kocak** [reply]
+1

**Comment by tryabp** [comment]
For reviews: It should be definitely improved! Integrating review plugin like TrustIndex (https://wordpress.org/plugins/free-facebook-reviews-and-recommendations-widgets/) is a great idea in terms of user experience and authentic reviews too.
